File: python_programming_basics/conditional_statements_advanced/l_11_fruit_shop.py
working_days_prices = {"banana": 2.5, "apple": 1.2, "orange": 0.85,
                       "grapefruit": 1.45, "kiwi": 2.7, "pineapple": 5.5, "grapes": 3.85}

weekend_days_prices = {"banana": 2.7, "apple": 1.25, "orange": 0.90,
                       "grapefruit": 1.6, "kiwi": 3.0, "pineapple": 5.6, "grapes": 4.2}

days = ["Monday", "Tuesday", "Wednesday", "Thursday", "Friday"]
weekend_days = ["Saturday", "Sunday"]
result = 0

fruit = input()
day = input()
quantity = float(input())

if fruit in working_days_prices and day in days:
    result = quantity * working_days_prices[fruit]
    print(f"{result:.2f}")

elif fruit in weekend_days_prices and day in weekend_days:
    result = quantity * weekend_days_prices[fruit]
    print(f"{result:.2f}")

else:
    print("error")

File: JavaScript Basics/03. Conditional Statements Advanced/l-11-Fruit-Shop.js
function calculateFruitPrice(fruit, day, quantity) {
    const weekdayPrices = {
        "banana": 2.50,
        "apple": 1.20,
        "orange": 0.85,
        "grapefruit": 1.45,
        "kiwi": 2.70,
        "pineapple": 5.50,
        "grapes": 3.85
    };

    
    const weekendPrices = {
        "banana": 2.70,
        "apple": 1.25,
        "orange": 0.90,
        "grapefruit": 1.60,
        "kiwi": 3.00,
        "pineapple": 5.60,
        "grapes": 4.20
    };

  
    if (!weekdayPrices[fruit] && !weekendPrices[fruit]) {
        console.log("error");
        return;
    }

    if (day === "Saturday" || day === "Sunday") {
        
        let price = weekendPrices[fruit] * quantity;
        console.log(price.toFixed(2));
    
    } else if (["Monday", "Tuesday", "Wednesday", "Thursday", "Friday"].includes(day)) {

        let price = weekdayPrices[fruit] * quantity;
        console.log(price.toFixed(2));
    
    } else {
 
        console.log("error");
    }
}

calculateFruitPrice("apple", "Tuesday", 2);
calculateFruitPrice("orange", "Sunday", 3);
calculateFruitPrice("kiwi", "Monday", 2.5);
calculateFruitPrice("grapes", "Saturday", 0.5);
calculateFruitPrice("tomato", "Monday", 0.5);